app/grandchallenge/evaluation/views/__init__.py

```diff
@@ -1252,3 +1255,94 @@ def get_object(self, queryset=None):
             challenge=self.request.challenge,
             slug=self.kwargs["slug"],
         )
+
+
+class AlgorithmInterfaceForPhasePermissionMixin(AccessMixin):
+    @property
+    def phase(self):
+        return get_object_or_404(
+            Phase,
+            challenge=self.request.challenge,
+            challenge__phase__submission_kind=SubmissionKindChoices.ALGORITHM,
+            challenge__phase__external_evaluation=False,
```

Comment:
Why are external evaluations excluded? I thought the interfaces had to match the parent phase for those?

Reply:
Oh you're right! I was under the impression that we shouldn't be configuring interfaces for external phases, but it's crucial that we can given that they need to match those of the parent phase.